homeassistant/components/lametric/notify.py

```diff
@@ -97,8 +97,12 @@ def send_message(self, message: str = "", **kwargs: Any) -> None:
                 icon = data["icon"]
             if "sound" in data:
                 try:
-                    sound = Sound(category="notifications", sound_id=data["sound"])
-                    LOGGER.debug("Adding notification sound %s", data["sound"])
+                    if "alarm" in data["sound"]:
```

Comment:
Although I am a fan of Lametric and this is nice, I do have a change pending for a complete rewrite of this integration.

Adding this, will cause this to become a breaking change later. For that reason, I'm not sure if we should add this right now.